daemon/cluster/cluster.go

```diff
@@ -468,11 +468,11 @@ func (c *Cluster) Join(req types.JoinRequest) error {
 	c.Lock()
 	if node := c.node; node != nil {
 		c.Unlock()
-		return ErrSwarmExists
+		return errors.NewRequestNotAcceptableError(ErrSwarmExists)
```

Comment:
Both `ErrNoSwarm` and `ErrSwarmExists` are always wrapped with `NewRequestNotAcceptableError`, maybe the variable should be a `NewRequestNotAcceptableError` ? so it doesn't have to get wrapped in each place it is used?